Formula/zld.rb

```diff
@@ -1,11 +1,14 @@
 class Zld < Formula
-  version "1.1.0"
+  version "1.2.1"
   desc "A faster version of ld, Apple's linker"
-  url "https://github.com/michaeleisel/zld/releases/download/#{version}/zld.zip"
-  sha256 "c606ec42d00b141576d2d2616ad3449a0ddca3edafd7917abdff063e919c8184"
+  url "https://github.com/michaeleisel/zld/archive/#{version}.zip"
+  sha256 "463faa979483a173d66c6204ce1289ddc05319ad70dfb0e685c1fbf9ca9ecd11"
```

Comment:
so the sha256 is now of the archive zip?

Reply:
Yes, calculated like this:

```sh
wget https://github.com/michaeleisel/zld/archive/1.2.1.zip
sha256sum 1.2.1.zip
# 463faa979483a173d66c6204ce1289ddc05319ad70dfb0e685c1fbf9ca9ecd11  1.2.1.zip
```